inference-engine/tests/functional/shared_test_classes/src/subgraph/split_concat_multi_inputs.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/subgraph/split_concat_multi_inputs.hpp"
+#include "ngraph_functions/builders.hpp"
+
+namespace SubgraphTestsDefinitions {
+
+std::string SplitConcatMultiInputsTest::getTestCaseName(testing::TestParamInfo<SplitConcatMultiInputsParams> obj) {
+    std::vector<size_t> inputShape;
+    size_t splitsNum;
+    std::map<std::string, std::string> config;
+    InferenceEngine::Precision netPrecision;
+    std::string targetName;
+    bool withFC;
+    std::tie(netPrecision, targetName, config, inputShape, splitsNum, withFC) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShape) << "_";
+    result << "SplitsN=" << splitsNum << "_";
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "trgDev=" << targetName << "_";
+    result << "FC=" << withFC;
+    for (auto const& configItem : config) {
+        result << "_configItem=" << configItem.first << "_" << configItem.second;
+    }
+    return result.str();
+}
+
+void SplitConcatMultiInputsTest::SetUp() {
+    std::vector<size_t> inputShape;
+    size_t splitsNum;
+    std::map<std::string, std::string> tempConfig;
+    InferenceEngine::Precision netPrecision;
+    //std::string targetDevice;
```

Comment:
Uncomment this. I'm wondering how it is working? It seems CI doesn't run GNA)

Reply:
targetDevice is initialized in base class LayerTestsCommon, so I will delete this